app/views/posts/show.html.erb

```diff
@@ -12,4 +12,7 @@
   
    <button><%= link_to "Add a comment", new_user_post_comment_path(current_user, @post),class:'add-comment' %></button>
    <%= button_to 'Like', user_post_likes_path(current_user, @post), method: :post%>
+    <% if can? :destroy, @post %>
+  <button>  <%= link_to 'Delete', user_post_path(current_user, @post),class:'add-comment', data: { turbo_method: :delete, turbo_confirm: 'Are you sure you want to delete this post?' }%></button>
+  <% end %>
 </div>
```

Comment:
- [ ] You have done well implementing the delete feature posts, but I cannot find the button for deleting a comment. Kindly ensure owners of comments and admins can see the delete button and can delete it.

  ![image](https://github.com/derrick1451/Blog-app/assets/36542986/02582787-791b-4814-9326-4452b74edc74)

Reply:
@TedLivist  a question is the comment button supposed to delete a single comment or all comments in a post

Reply:
@derrick1451 the button should delete the comment that is being deleted, not all comments. So a single comment.

Reply:
So if there are multiple comments on a post, each comment should have its own delete button - of course depending on the user logged in.